devices/HumanStateProvider/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@
 
 find_package(IWear REQUIRED)
 find_package(iDynTree REQUIRED)
+find_package(OsqpEigen 0.4.0 REQUIRED)
```

Comment:
Is there a specific reason why you require version `0.4.0`? I see the current [`osqp-eigen` master](https://github.com/robotology/osqp-eigen/blob/master/CMakeLists.txt#L30) is on `0.3.0`

Reply:
This no longer apply after https://github.com/robotology/osqp-eigen/commit/ac2a55166e5b72f9d092e5988b1b4a0c7f7d6554.